package.json

```diff
@@ -2,17 +2,21 @@
   "name": "fastify-enterprise",
   "version": "1.0.0",
   "description": "",
-  "type": "module",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node index.js",
-    "dev": "nodemon index.js"
+    "dev": "nodemon index.js",
```

Comment:
Remove unused scripts and update build, dev and start script with pino prettier as well